### PR TITLE
[MO] [TF] Added TF broadcast_to extractor

### PR DESCRIPTION
**Pull request by pavel-esir**
Description: Added TF broadcast_to extractor

Ticket: 40372

Code:
* [x]  Comments
* [x]  Code style (PEP8)
* [x]  Transformation generates reshape-able IR
* [x]  Transformation preserves original framework node names

Validation:
* [x]  Unit tests: N/A
* [x]  Framework operation tests
* [x]  Transformation tests: N/A
* [x]  Model Optimizer IR Reader check

Documentation:
* [x]  Supported frameworks operations list
* [x]  Guide on how to convert the **public** model: N/A
* [x]  User guide update: N/A

### CODE REVIEW

File: model-optimizer/extensions/front/tf/broadcast_ext.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from mo.front.extractor import FrontExtractorOp
+from mo.graph.graph import Node
+from mo.ops.broadcast import Broadcast
+
+
+class SliceExtractor(FrontExtractorOp):
```
Comment (lazarevevgeny) [inline model-optimizer/extensions/front/tf/broadcast_ext.py:22]: Change extractor name

**Review by lazarevevgeny — CHANGES_REQUESTED** [review]

File: model-optimizer/extensions/front/tf/broadcast_ext.py (hunk 1 above)
Comment (pavel-esir) [reply model-optimizer/extensions/front/tf/broadcast_ext.py]: done

**Review by iimironov — APPROVED** [review]

File: docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md [hunk 2]
```diff
@@ -134,6 +134,7 @@ Standard TensorFlow\* operations:
 | BatchToSpaceND | No |
 | BiasAdd | No |
 | Bucketize | CPU only |
+| Broadcast_to| No |
```
Comment (lazarevevgeny) [inline docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md:137]: Change to `BroadcastTo`. We use TF op low-level names in this table (not python API names)

Comment (pavel-esir) [reply docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md]: done

**Review by lazarevevgeny — APPROVED** [review]

**Review by evolosen — APPROVED** [review]

**Pull request merged**